package.json

```diff
@@ -64,6 +64,7 @@
   "homepage": "https://github.com/software-mansion/react-native-gesture-handler#readme",
   "dependencies": {
     "@egjs/hammerjs": "^2.0.17",
+    "fast-xml-parser": "^4.4.1",
```

Comment:
Is it necessary?

Reply:
it's not, removed in [4c12996](https://github.com/software-mansion/react-native-gesture-handler/pull/3055/commits/4c1299690a915c6258c0bf67c0e3a6f14ed9db29)